java/src/main/java/com/saucelabs/saucebindings/SystemManager.java

```diff
@@ -0,0 +1,29 @@
+package com.saucelabs.saucebindings;
+
+public class SystemManager {
+    protected static String get(String key, String errorMessage) {
```

Comment:
@titusfortner are these static methods going to be a problem when it comes to parallelization?

Reply:
Not with the way surefire works, but potentially with TestNG, but that's why people need to use thread local. 

That said, it would make sense to test this out with an example suite before getting too far along.

Reply:
I'm going to guess this is fine since these methods only return "constant" string values. These values should remain constant across tests/classes anyway.

Reply:
Oh, I was looking at wrong code, yes, there is no state needed for these, so no need to instantiate anything.